typescript/sdk/src/core/HyperlaneCore.ts

```diff
@@ -121,6 +127,38 @@ export class HyperlaneCore extends HyperlaneApp<CoreFactories> {
     }
   }
 
+  async send(
```

Comment:
sendMessage

Reply:
Will you be updating the CLI to use this instead? Also would be great to see this in MultiProtocolCore

Reply:
https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/typescript/cli/src/send/message.ts#L105